api/src/controllers/ticket_types.rs

```
@@ -249,6 +257,13 @@ pub fn update(
     jlog!(Debug, "Updating ticket type", {"ticket_type_id": path.ticket_type_id, "event_id":event.id, "request": &data});
     let ticket_type = TicketType::find(path.ticket_type_id, connection)?;
     if let Some(requested_capacity) = data.capacity {
+        //Check that requested ticket capacity is less than max_tickets_per_ticket_type
+        if requested_capacity as i64 > organization.max_tickets_per_ticket_type {
+            return application::internal_server_error(
```

It's better to return a validation error. An internal server error will appear in the logs as a error rather than a warning

---

Changed to application::unprocessable error